array-methods/map.js

```diff
@@ -0,0 +1,23 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+console.log('Price objects');
+console.log(
+  prices.map((element) => {
+    const newArr = [];
+    newArr.push({ price: element, salePrice: element / 2 });
+    return newArr;
+  })
+);
+
+console.log('Formatted Prices ');
+console.log(
+  prices.map((a) => {
+    let formatted = '';
```

Comment:
There's no reason to declare this as an empty string and then concatenate to it. You can just say `const formatted = new Intl.....`;
In fact, you could simplify it further with
`prices.map((a) => new Intl.NumberFormat(...).format(a));`